src/staking-v3/logic/services/DappStakingService.ts

```diff
@@ -61,6 +61,12 @@ export class DappStakingService implements IDappStakingService {
     return { fullInfo: dApps, chainInfo: onlyChain };
   }
 
+  public async getNumberOfParticipants(network: string): Promise<number> {
```

Comment:
Small comment. Having a service method which just wraps a repo method can be omitted if there is no additional logic in service layer.